pkg/api/defaults_test.go

```diff
@@ -2947,44 +2947,104 @@ func TestDefaultEnablePodSecurityPolicy(t *testing.T) {
 			expected: false,
 		},
 		{
-			name: "default",
+			name: "default, >= 1.15",
 			cs: ContainerService{
 				Properties: &Properties{
 					OrchestratorProfile: &OrchestratorProfile{
 						OrchestratorType:    Kubernetes,
-						OrchestratorVersion: "1.15.0-alpha.1",
```

Comment:
Thanks for cleaning these -alpha and -beta references up. 

There are similar code droppings related to Kubernetes 1.16.0-(alpha,beta,rc). Should we make a PR to have those point to the release as well?